core/src/main/java/org/hibernate/search/jsr352/internal/steps/lucene/EntityReader.java

```diff
@@ -242,7 +242,8 @@ private ScrollableResults buildScrollUsingHQL(StatelessSession ss, String HQL) {
 	private ScrollableResults buildScrollUsingCriteria(StatelessSession ss,
 			PartitionBound unit, Object checkpointId, JobContextData jobData) {
 
-		String idName = MassIndexerUtil.getIdName( entityType, session );
+		String idName = Projections.id().toString();
```

Comment:
So basically, I don't see how to use Alias and I don't know how to change the code. Could you give me more detail, ideally some lines of code?